scripts/vote.ts

```diff
@@ -91,16 +87,13 @@ const app = command({
 
     const nouns_descriptor: nouns_contract.NounsContractDescriptor = JSON.parse(
       fs.readFileSync(nc_descriptor_file, 'utf8'));
-    expect(my_id).is.lessThanOrEqual(dkg_descriptor.n_comm);
 
-    // Connect
-    const provider = new ethers.providers.JsonRpcProvider(endpoint);
+    const password = process.env.KEYFILE_PASSWORD;
```

Comment:
What happens if this is not set?  Does it crash, or use the empty string?

Reply:
Ok, I tested it out and it crashes. So I will add some code so it functions more gracefully.

Reply:
Ok done.